### ISSUE TITLE
Extracting calculated word_attributions from predictor.explain() method

### ISSUE THREAD

**Issue by dtg-ca** [amaiya/ktrain#397]
Hi Arun, 
           First of all thank you for producing and maintaining this amazing library. It makes it possible for newbies like me to attempt and solve some challenging problems with the help of Data science & Machine learning. 
I am doing a binary text classification to predict 2 classes, response_needed & no_response on a dataset of collected emails. I want to look at the calculated output of individual word contributions in the text from the predictor.explain() method. I can hover over the words and see that the calculated weights are available., but do not know how to extract them as ( word, weight) from this method. Your help to unlock this would be greatly appreciated. Thanks again!

**Comment by amaiya** [comment]
Thanks for your comments.

The `TextPredictor` explain method currently returns an HTML object representing the explanation.  You can parse the HTML to get anything you want with something like this:

```python
p = ktrain.get_predictor(model, preproc)
some_text = 'Jesus Christ is the central figure of Christianity.'
html = p.explain(some_text)
from bs4 import BeautifulSoup
soup = BeautifulSoup(html.data)
for span in soup.find_all('p')[1].find_all('span'):
    if not span.get('style').startswith('background-color'): continue
    word = span.contents[0]
    weight = float(span.get('title'))
    print(word, weight)
# output
# jesus 0.332
# christ 0.428
# is -0.077
# central 0.311
# figure 0.027
# of 0.046
# christianity 1.313
```

The `TextPredictor.explain` method uses the `eli5` library.  There's probably an `eli5` method that will allow you to access the word weights directly, if you peruse their documentation.

**Comment by dtg-ca** [comment]
Awesome, so grateful for your prompt response. This is exactly what I was looking for ...thank you!!!. I will also check out the eli5 library as per your suggestion.
